rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/20_unsupported_apis.yml

```diff
@@ -61,7 +61,6 @@
         index: logs-foobarbaz
 
   - do:
-      catch: bad_request
```

Comment:
This fails when this yaml test is executed with security enabled.
`logs-*` is expanded to `logs-foobar` and then later this fails with a 404.

So in order to make this test execute constantly with and without security enabled, we should change 
`index: logs-*` to `index: logs-foobar. This should change the meaning of this test.